src/main/resources/api.yaml

```diff
@@ -235,3 +298,21 @@ definitions:
     required:
       - enrollmentPeriod
       - durationPeriod
+
+  AvailabilityDto:
+    type: object
+    properties:
+      userId:
+        type: integer
+        format: int64
+      timeAvailable:
+        type: integer
```

Comment:
Z tego się robi `Duration`, nie? Tutaj mamy longi w drugim przypadku z `Duration`

Reply:
`Duration.ofMinutes` przyjmuje longa

Reply:
Aaa, `timeBurned` i `timePlanned` w sprincie to są

Reply:
te nazwy chyba nie muszą mieć ze sobą nic wspólnego?

Reply:
Nie nazwy - to `timeBurned` i `timePlanned` są w `Sprint`, nie w `Availability` - po prostu nie mogłam sobie przypomnieć, gdzie to było (a bardzo mi się nie chce otwierać IntelliJa) i byłam dumna, że sobie przypomniałam

Chodziło mi tylko o typ danych